### DISCUSSION TITLE
Welcome to Thawani Pay WooCommerce Discussions!

### DISCUSSION THREAD

**Discussion by malnafei** [malnafei/thawani-pay-woocommerce#1] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by w7shdev** [comment]
Hi,  thanks for the effort 
I've developed a commercial version of the integration myself. however , I saw there is area of improvement to the code and we can make it more maintainable.

**Reply by malnafei** [reply]
Hello @w7shdev 
You're welcome.
Please feel free to create new issues or pull requests anytime.

**Reply by w7shdev** [reply]
No worries I'm thinking of creating a new PR for replacing a php curl with the build in curl function in wordpress such as https://developer.wordpress.org/reference/functions/wp_remote_get/  and `wp_remote_post` it will be handy for unit testing and ERROR handling for wordpress
